framework/src/util/Properties.h

```diff
@@ -36,24 +36,22 @@ class Properties : public detail::MultiThreaded<>
 {
 
 public:
+  explicit Properties(AnyMap& props);
```

Comment:
Is this constructor necessary? If it doesn't actually modify props, it can be removed... the const& constructor will be called instead.

Reply:
Not necessary, removed.